terraform/modules/ssm/parameters.tf

```diff
@@ -0,0 +1,31 @@
+# Resource - specific parameters are to be created in their respective modules
+
+resource "aws_ssm_parameter" "one_login_public_key" {
+  name  = "${var.environment_name}-one-login-public-key"
+  type  = "String"
+  value = "default" # To be set manually on AWS
```

Comment:
Extremely minor thought: instead of `default` could we use something like `default_to_be_manually_set`? Just in case someone forgets to do it or something goes wrong and TF overwrites the value or something. Might make debugging fractionally easier?